test/E2E/E2E.xproj

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Acutally no, it looks like the other one is wrong. VS expects the xproj to be named after the folder name I believe. I'll remove the other one. It shouldn't cause a problem.